docs/conf.py

```diff
@@ -46,7 +46,15 @@
 # files
 templates_path = ["_templates"]
 html_favicon = "../app/lab/favicon.ico"
-html_static_path = ["_static", "../app"]
+# rely on the order of these to patch json, labextensions correctly
+html_static_path = [
+    # docs stuff
+    "_static",
+    # the as-built application
+    "../app",
+    # extensions and patched jupyter-lite.json
+    "../build/env-extensions",
```

Comment:
this is now a bit more "normal means," and we let the path order take care of it